src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -229,6 +231,28 @@ public static String getListDefaultDatabaseValue() {
 
     private final boolean standalone;
 
+    private final Function<ProductContentInfo, ProductContent> productContentInfoToProductContent =
+        (pcinfo) -> {
+            ContentInfo cinfo = pcinfo.getContent();
+
+            org.candlepin.model.Content converted = new org.candlepin.model.Content();
```

Comment:
nit: we can use the fluent nature of the mutators on Content to avoid repeating the `converted.` all the way down.